### PR TITLE
Standardize buildtime

### PR DESCRIPTION
**Pull request by thehobojoe**
Adjusts the buildtime of nearly every unit, to simplify very messy values like `27982` to `28000`, etc. I set a firm limit of an adjustment of 0.5% in any direction. For economy buildings and for meta-heavy bread-and-butter units like grunts, thugs, stouts, etc, I limited the adjustment to about 0.1% to avoid any noticeably game-changing outcomes.

Whether the adjustment was an increase or decrease is fairly random, depending on how close the nearest "nice" number was in either direction, this should also help avoid any cumulative changes having noticeable outcomes in-game.

### CODE REVIEW

File: units/ArmAircraft/T2/armlance.lua [hunk 1]
```diff
@@ -5,7 +5,7 @@ return {
 		buildcostenergy = 8000,
 		buildcostmetal = 400,
 		buildpic = "ARMLANCE.DDS",
-		buildtime = 15096,
+		buildtime = 15100,
```
Comment (sprunk) [inline units/ArmAircraft/T2/armlance.lua:8]: 15000?

File: units/ArmBots/armrock.lua [hunk 2]
```diff
@@ -5,7 +5,7 @@ return {
 		buildcostenergy = 1000,
 		buildcostmetal = 120,
 		buildpic = "ARMROCK.DDS",
-		buildtime = 2015,
+		buildtime = 2010,
```
Comment (sprunk) [inline units/ArmBots/armrock.lua:8]: 2000? still under 1%

File: units/ArmBuildings/LandDefenceOffence/armemp.lua [hunk 3]
```diff
@@ -6,7 +6,7 @@ return {
 		buildcostenergy = 29000,
 		buildcostmetal = 1600,
 		buildpic = "ARMEMP.DDS",
-		buildtime = 79247,
+		buildtime = 79300,
```
Comment (sprunk) [inline units/ArmBuildings/LandDefenceOffence/armemp.lua:9]: 80000? still under 1%

File: units/ArmBots/armrock.lua (hunk 2 above)
Comment (thehobojoe) [reply units/ArmBots/armrock.lua:8]: missed this before, but this one in particular I didn't want to move much as it's a very staple earlygame unit - I want nice values, but definitely not at the expensive of broader balance. I restricted myself to 0.1% for these types of units.

**Pull request merged**